pkg/credentials/iam_aws.go

```diff
@@ -281,6 +277,10 @@ func fetchIMDSToken(client *http.Client, endpoint string) (string, error) {
 // If the credentials cannot be found, or there is an error
 // reading the response an error will be returned.
 func getCredentials(client *http.Client, endpoint string) (ec2RoleCredRespBody, error) {
+	if endpoint == "" {
+		endpoint = defaultIAMRoleEndpoint
+	}
```

Comment:
can you share the document where it says defaultIAMRoleEndpoint is the default for IMDS token?

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html

Reply:
ah I see its same endpoint.. for newer feature as well.